blocks/hero/hero.css

```diff
@@ -0,0 +1,76 @@
+/* block specific CSS for hero banner goes here */
+
+main .hero-container>div {
+    max-width: unset;
+  }
+  
+  main .only-picture{
+    display: none;
+  }
+  
+  main .hero-container {
+    padding: 0;
+  }
+  
+  main .hero {
+    position: relative;
+    max-width: 327px;
+    margin: 0 auto;
+  }
+  
+  main .hero h1 {
+    color: white;
+    font-size: 22px;
+    margin: 0;
+    font-family: var(--sans-serif-font-regular);
+    font-style: normal;
+    font-weight: var(--font-weight-regular);
+  }
+  
+  main .hero img {
+    width: 100%;
+    height: 100%;
+    border-radius: 10px;
+    border: 1px solid transparent;
+    display: block;
+  }
+  
+  main .hero #logo-text-holder {
+    position: absolute;
+    bottom: 24px;
+    margin-left: 24px;
+  }
+  
+  main .hero #logo-text-holder svg {
+    width: 35.23px;
+    height: 35.23px;
+  }
+  
+  /* Tablet */
+  @media (min-width: 768px) {
+   main .hero{
+      max-width: 696px;
+      }
+    main .hero h1 {
+      font-size: var(--font-size-28);
+    }
+    main .hero #logo-text-holder {
```

Comment:
Avoid using Id in CSS selectors.

Reply:
@sahmad-merative - The requested changed have been done. Please have a look.